src/config.c

```diff
@@ -2601,6 +2601,61 @@ int allowProtectedAction(int config, client *c) {
            (config == PROTECTED_ACTION_ALLOWED_LOCAL && islocalClient(c));
 }
 
+
+static int setConfigLatencyPercentileOption(typeData data, sds *argv, int argc, const char **err) {
+    UNUSED(data);
+
+    /* Special case: treat single arg "" as zero args indicating empty percentile configuration */
+    if (argc == 1 && !strcasecmp(argv[0],""))
+        argc = 0;
+
+    resetServerLatencyPercentileParams();
+
+    for (int j = 0; j < argc; j++) {
+        double percentile;
+        if (!string2d(argv[j], sdslen(argv[j]), &percentile)){
+            *err = "Invalid latency-track-percentiles parameters";
```

Comment:
We should validate all the percentiles before starting to apply them. We don't want them half applied.

Reply:
it might be a good idea, but IIRC, the config infra handles rollback now. @yoav-steinberg correct me if i'm wrong.

Reply:
@oranagra I've added a specifc test to verify that indeed the config is rolled back it's in `latencystats: bad configure percentiles`.